32-Toggle CheckBox/script.js

```diff
@@ -0,0 +1,24 @@
+const toggles = document.querySelectorAll('.toggle')
+const good = document.querySelector('#good')
+const cheap = document.querySelector('#cheap')
+const fast = document.querySelector('#fast')
+
+toggles.forEach(toggle => toggle.addEventListener('change', (e) => {
+  cantHaveItAll(e.target)
+}))
```

Comment:
Okay I think we've already talked about this issue/behaviour multiple times now!!
why creating the wrapping arrow function if we are just using it to invoke another function ??? seems like waste of resources for me! the only gain here seem to be being able to invoke `cantHaveItAll` with the `e.target` which could've easily be done inside `cantHaveItAll` it self.

ex. 
```
toggles.forEach(toggle => toggle.addEventListener('change', cantHaveItAll);
...
...
function cantHaveItAll(e) {
 const theClickedOne = e.target;
  ...
  ...
}

```